### PR TITLE
Introduces 'MOCK_COUNTERVALUES' to test countervalues in MOCK mode

### PR DESCRIPTION
**Pull request by gre**
### 📝 Description

there are no e2e coverage of countervalues api and we might easily break it if we are not careful. This introduces a simple e2e that verifies in playwright that a http call occurred with our countervalues api.

for that, i needed to split the MOCK_COUNTERVALUES concept out of the MOCK.
generally speaking it's a good practice to start splitting MOCK into many different "concerns".

### ❓ Context

- **Impacted projects**: `n/a` 
- **Linked resource(s)**: `n/a` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/tests/specs/countervalues/api.spec.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import test from "../../fixtures/common";
+import { expect } from "@playwright/test";
+
+test.use({ userdata: "1AccountBTC1AccountETH" });
+
+test.use({
+  env: {
+    // disable mock for countervalues
+    MOCK_COUNTERVALUES: "",
```
Comment (hedi-edelbloute) [inline apps/ledger-live-desktop/tests/specs/countervalues/api.spec.ts:9]: Why is it a boolean in the file above (common.ts) and an empty string here. Shouldn't it be false ?

Comment (gre) [reply apps/ledger-live-desktop/tests/specs/countervalues/api.spec.ts:9]: indeed it's weird that it was sometimes set a value to `true`, I basically did exactly like we do for `MOCK`, in MOCK case, the env is a string that is the "seed" of the mock generator. we could have same principle on MOCK_COUNTERVALUES even tho today it's only used as a on/off.
the `true` is going to actually be a `"true"` seed 😆 
anyway, nothing i really changed on the MOCK paradigm, as you can see it's parsed with `stringParser` like for the env MOCK

Comment (elbywan) [reply apps/ledger-live-desktop/tests/specs/countervalues/api.spec.ts:9]: As far as I know, environment variables are always strings so I would say it does not matter at all and true gets stringified to "true".

Comment (gre) [reply apps/ledger-live-desktop/tests/specs/countervalues/api.spec.ts:9]: live common env.ts still have some sort of comprehension of string->boolean, basically will check if it's "true" or "false" to match to booleans **if** an env is defined as boolean. which is not the case of MOCK_COUNTERVALUES atm

Comment (elbywan) [reply apps/ledger-live-desktop/tests/specs/countervalues/api.spec.ts:9]: ah good to know 👍

Comment (hedi-edelbloute) [reply apps/ledger-live-desktop/tests/specs/countervalues/api.spec.ts:9]: Ok thanks :)

**Review by hedi-edelbloute — APPROVED** [review]

**Pull request merged**